src/libraries/System.CodeDom/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,634 @@
+<?xml version="1.0"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <!-- These types are forwarded on net461 https://github.com/dotnet/sdk/issues/18203-->
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
```

Comment:
Can we instead use the global suppression? Rather than listing every forwarded type on net461, instead set one entry with the DiagnosticId and the left and right?

Reply:
If that doesn't work, maybe we should support it.

Reply:
We can now simplify these as we have updated the validation to include: https://github.com/dotnet/sdk/pull/19123